extra/desktop/Anarchy-gnome/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/prefs.js/ui/prefs.ui

```diff
@@ -19,6 +19,7 @@ Mosaab Alzoubi (moceap)
 Alonso Lara (AlonsoLP)
 Dingzhong Chen (wsxy162)
 narzb
+Remisa Yousefvand (yousefvand)
```

Comment:
Same here

Reply:
I was not sure about this one. By removing the other one, there would be nowhere to add translator, while instructions told add yourself. I'd leave it up to your team.
BWT I'd like to work on using docker as ISO building incubator (without `Archiso` at this step). Is it OK?

Reply:
You did add yourself in the anarchy-persian.conf. Sorry if the instructions weren't clear enough.
By add yourself I meant add your name and email in the translation file, which you did properly.

Reply:
> BWT I'd like to work on using docker as ISO building incubator (without `Archiso` at this step). Is it OK?

Of course. I'd prefer if the Dockerfile is in the root directory of the repo, so it's easier for new users to find, it that seems reasonable to you.

Reply:
Sure I'd make it in project root and update you with result.

Reply:
Thanks